actionners/kubernetes/log/log.go

```diff
@@ -28,17 +30,21 @@ func Action(action *rules.Action, event *events.Event) (utils.LogLine, error) {
 	}
 
 	parameters := action.GetParameters()
-	tailLines := new(int64)
-	if parameters["tail_lines"] != nil {
-		*tailLines = int64(parameters["tail_lines"].(int))
-	}
-	if *tailLines == 0 {
-		*tailLines = 20
+	var config Config
+	err := utils.DecodeParams(parameters, &config)
+	if err != nil {
+		return utils.LogLine{
+			Objects: nil,
+			Error:   err.Error(),
+			Status:  "failure",
+		}, nil, err
 	}
 
-	command := new(string)
-	if parameters["command"] != nil {
-		*command = parameters["command"].(string)
+	tailLines := new(int64)
+	if config.TailLines == 0 {
```

Comment:
can we have 20 as a constant and added to the docs as default?

Reply:
As of now default is 1000, am I missing something here? 
https://docs.falco-talon.org/docs/actionners/list/#kuberneteslog

In the code, even before it was 20 already.

Reply:
The docs is wrong, let me create the constant and I'll fix the docs after.